test/core/utils/respondable.js

```diff
@@ -112,40 +112,28 @@ describe('axe.utils.respondable', function() {
       });
     });
 
-    it('should allow messages with _source axeAPI.x.y.z', function(done) {
+    it('should reject messages if the axe version is different', function(done) {
+      axe.version = '1.0.0';
```

Comment:
Should probably undo this after the test is done.

Reply:
Tests already setup for that https://github.com/dequelabs/axe-core/blob/b364c451bf14d742ee5272c248aefde364dfc160/test/core/utils/respondable.js#L97-L99